ajax/libs/overpass/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "overpass",
+  "version": "3.0.2",
+  "filename": "overpass-webfont/overpass.css",
```

Comment:
@sufuf3 Yes, I've added minified versions for all CSS files in all the versions 😄